src/graphql/company.js

```diff
@@ -254,13 +254,20 @@ export const getCompanyEsgSalaryDataQuery = /* GraphQL */ `
 `;
 
 export const getCompanyInterviewExperiencesQuery = /* GraphQL */ `
-  query($companyName: String!, $jobTitle: String, $start: Int!, $limit: Int!) {
+  query(
+    $companyName: String!
+    $jobTitle: String
+    $start: Int!
+    $limit: Int!
+    $sortBy: SortOption
```

Comment:
```suggestion
    $sortBy: DataResultSortOption
```

API will failed with below message
```
Unknown type \"SortOption\". Did you mean \"Duration\", \"Mutation\", or \"Section\"?
```

Need to change `SortOption to` `DataResultSortOption`